src/test/java/by/teachmeskills/LoginTest.java

```diff
@@ -0,0 +1,36 @@
+package by.teachmeskills;
+
+import by.teachmeskills.page.Header;
+import by.teachmeskills.page.LoginPage;
+import by.teachmeskills.page.ProjectsPage;
+import by.teachmeskills.step.LoginSteps;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LoginTest {
+    @Test
+    public void loginAsValidUser() {
+        new LoginSteps().loginWithValidCredentials();
+        assertThat(new ProjectsPage().isOpened())
+                .as("After entering valid credentials Projects page should be opened")
+                .isTrue();
+        new Header().signOut();
+    }
+
+    @Test
+    public void loginWithEmptyValues() {
+        new LoginSteps().login("", "");
```

Comment:
jfyi: https://stackoverflow.com/questions/73283699/how-to-find-prompt-by-selenium